sdcm/cluster.py

```diff
@@ -2773,6 +2773,12 @@ def is_replacement_by_host_id_supported(self):
             return ComparableScyllaVersion(self.scylla_version) > '2022.3.0~dev'
         return ComparableScyllaVersion(self.scylla_version) > '5.2.0~dev'
 
+    @property
+    def is_cluster_have_system_auth_ks(self):
+        if self.is_enterprise:
+            return ComparableScyllaVersion(self.scylla_version) <= '2024.1'
```

Comment:
I think this can check raft topology, since it's coming with it

Reply:
Its raft topology changes, I'm not sure what exactly is_raft checks

Reply:
Okay, I'll double-check it, perhaps, I used this check mistakenly

Reply:
> I think this can check raft topology, since it's coming with it

Is that the case? Even when raft topology/schema was enabled in Scylla 5.4, system_auth was still there.
I thought that it's a change introduced in 6.0 that requires raft schema - but in 6.0 it's impossible to disable raft schema so it's more about the version than configuration.

Reply:
There is one more observation in addition to Michal's reply.
If to query Scylla enabled_features for:
- Scylla 5.4 with enabled `consistent-topology-changes` - no `SUPPORTS_CONSISTENT_TOPOLOGY_CHANGES` among the features;
- Scylla 6.0 - `SUPPORTS_CONSISTENT_TOPOLOGY_CHANGES` presents there.

I believe we can use it as a definition point.

Reply:
Yes auth_v2 goes hand by hand with raft topology changes.

And it can be disabled with force_gossip (not the exact name, I don't remember it)

So I still recommend checking scylla features when possible, it's a better option then counting on versions (since feature can be enabled/disabled in all bunch of ways)

Reply:
Yes, I fixed it and retested here:
- Scylla 2024.1 - https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-sanity-test/21/
- Scylla 6.0 - https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-vnodes-tablets-cluster-test/11/